src/main/java/com/lambdatest/jenkins/freestyle/api/service/AppAutomationCapabilityService.java

```diff
@@ -0,0 +1,164 @@
+package com.lambdatest.jenkins.freestyle.api.service;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+
+import org.apache.commons.collections.MapUtils;
+import org.json.JSONObject;
+import org.springframework.util.CollectionUtils;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.lambdatest.jenkins.freestyle.api.Constant;
+import com.lambdatest.jenkins.freestyle.api.device.Devices;
+import com.lambdatest.jenkins.freestyle.api.device.Device;
+import com.lambdatest.jenkins.freestyle.api.device.DeviceVersion;
+import com.lambdatest.jenkins.freestyle.api.operatingsystem.OsList;
+
+public class AppAutomationCapabilityService {
+
+    private final static Logger logger = Logger.getLogger(CapabilityService.class.getName());
```

Comment:
Please change `CapabilityService.class` to `AppAutomationCapabilityService.class`

Reply:
Done